gitlab/v4/objects/issues.py

```diff
@@ -55,8 +55,10 @@ class IssueManager(RetrieveMixin, RESTManager):
         "state",
         "labels",
         "milestone",
+        "iteration_id",
         "scope",
         "author_id",
+        "iteration_id",
```

Comment:
Looks like this has been done twice. L58 above already does this.

Reply:
whops, thanks for finding it!

Reply:
I just removed the duplication!